test/unit/tsx/input.tsx

```diff
@@ -5,4 +5,4 @@ function Tsx() {
 }
 
 export default Tsx
-export { type FooString } from './foo'
```

Comment:
Shouldn't contain test changes

Reply:
I ran `pnpm format`, I'll revert it, my bad.

Reply:
I honestly don't know what change was made here 😅.